Source/FileSystem.Injectable.cpp

```diff
@@ -23,6 +23,11 @@ namespace vl
 			return injectedFileSystemImpl ? injectedFileSystemImpl : GetOSFileSystemImpl();
 		}
 
+		IFileSystemImpl* GetDefaultFileSystemImpl()
```

Comment:
It is completely wrong. If you can't run my makefile, you won't find the actual symbol that is a "undefined reference".